test/rust_src/src/buffers-tests.el

```diff
@@ -7,11 +7,13 @@
 (ert-deftest test-buffer-base-buffer-indirect ()
   (let* ((base (get-buffer-create "base"))
          (ind-buf (make-indirect-buffer base "indbuf")))
-    (should (eq (buffer-base-buffer ind-buf) base))))
+    (should (eq (buffer-base-buffer ind-buf) base))
```

Comment:
I've tried to make all the tests self-contained - previously, they weren't so tests were relying on state from other tests. Knowing what buffers are present is pretty much needed to test `other-buffer` cleanly.